src/components/Search/SearchProjects.js

```diff
@@ -8,65 +10,131 @@ import ProjectIcon from './../ProjectIcon'
-          ? hasAssetById({ listItems: watchlistItems, id })
-          : false
-        return (
-          <div className={styles.projectWrapper}>
-            <div className={styles.projectInfo}>
-              {isCopyingAssets ? (
-                <Checkbox
-                  isActive={checkedAssets.has(id)}
-                  className={styles.checkbox}
-                />
-              ) : (
-                <ProjectIcon className={styles.icon} size={16} name={name} />
-              )}
-              <span className={styles.name}>{name}</span>
-              <span className={styles.ticker}>({ticker})</span>
-            </div>
-            {isEditingWatchlist && (
-              <Icon
-                fill={`var(--${isAssetInList ? 'casper' : 'jungle-green'}`}
-                type={isAssetInList ? 'remove' : 'plus-round'}
-              />
-            )}
-          </div>
-        )
-      }}
+      withMoreSuggestions={false}
+      data={[
+        {
+          predicate: assetsPredicate,
+          title: 'Assets',
+          items: projects,
+          suggestionContent: ({ name, ticker, id }) => {
+            const isAssetInList = isEditingWatchlist
+              ? hasAssetById({ listItems: watchlistItems, id })
+              : false
+            return (
+              <div className={styles.projectWrapper}>
+                <div className={styles.projectInfo}>
+                  {isCopyingAssets ? (
+                    <Checkbox
+                      isActive={checkedAssets.has(id)}
+                      className={styles.checkbox}
+                    />
+                  ) : (
+                    <ProjectIcon
+                      className={styles.icon}
+                      size={16}
+                      name={name}
+                    />
+                  )}
+                  <span className={styles.name}>{name}</span>
+                  <span className={styles.ticker}>({ticker})</span>
+                </div>
+                {isEditingWatchlist && (
+                  <Icon
+                    fill={`var(--${isAssetInList ? 'casper' : 'jungle-green'}`}
+                    type={isAssetInList ? 'remove' : 'plus-round'}
+                  />
+                )}
+              </div>
+            )
+          }
+        },
+        {
+          predicate: trendWordsPredicate,
+          title: 'Trending words',
+          items: trendWords,
+          suggestionContent: word => word
+        }
+      ]}
     />
   )
 }
 
-export default graphql(allProjectsForSearchGQL, {
-  skip: ({ projectsList }) => projectsList && projectsList.length > 0,
-  options: () => ({
-    context: { isRetriable: true },
-    variables: { minVolume: 0 }
+const enhance = compose(
+  graphql(TRENDING_WORDS_QUERY, {
+    options: () => {
+      const from = new Date()
+      const to = new Date()
+      to.setHours(to.getHours(), 0, 0, 0)
+      from.setHours(from.getHours() - 8, 0, 0, 0)
```

Comment:
Why do we need to set up 8 hours? Not 1? @DmitriVanGuard

Reply:
Similar approach that we have used for the `Last trends`. But sure, we can make them "more" up-to-date in a search box. I will change it to the `1 hour`